recipes/pkg-config/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class PkgConfigConan(ConanFile):
+    name = "pkg-config"
+    description = "The pkg-config program is used to retrieve information about installed libraries in the system"
+    topics = ("conan", "pkg-config", "package config")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/pkg-config/"
+    license = "GPL-2.0-or-later"
+    settings = "os_build", "arch_build", "compiler"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os_build == "Windows" and self.settings.compiler == "gcc" and os.name == "nt"
+
+    def build_requirements(self):
+        if self._is_mingw_windows and "CONAN_BASH_PATH" not in os.environ:
```

Comment:
This will need an additional check for a system env var

Reply:
I don't understand what env var you are talking about. I can add ` and os_info.detect_windows_subsystem() != 'msys2'`

Reply:
yes, that was it